### DISCUSSION TITLE
Water level applied on one side of embankment only

### DISCUSSION THREAD

**Discussion by genclub33** [Anura3D/Anura3D_OpenSource#162] [Q&A]
Dear all,

I would like to apply the water level to only one side of the embankment (as shown in the figure) and use MPM to reproduce the seepage flow in the embankment. Is it possible to consider such a situation with Anura3D? In this case, how should I apply the hydraulic boundary condition (x, y min and max of hydraulic head and seepage face)?
And I am thinking of using unsaturated material- 2 phase with suction effect for the soil.
Sorry to ask such a basic question, but I look forward to hearing.

Sincerely,

![embankment](https://github.com/Anura3D/Anura3D_OpenSource/assets/114758814/32f505e6-3750-4381-a5c9-27f6b0710cc0)

**Comment by Abbki** [comment]
I tried the below boundary condition and seems to be working. I haven't verified the results yet.
![Screenshot 2024-01-19 181342](https://github.com/Anura3D/Anura3D_OpenSource/assets/156711964/3eea4d90-a25d-47f1-84bd-abb1461953aa)
